ebmlite/util.py

```diff
@@ -31,6 +31,56 @@
 # ==============================================================================
 
 
+def createID(idClass, schema, exclude=(), min=0x80, max=0x1FFFFFFF, count=1):
```

Comment:
tests for this?

Reply:
Good point, I'll do those in a bit.

Reply:
Tests added in commit ebf540431cee302d6023d4dabfcb6f5906d64c4f.